test/customValidators_test.js

```diff
@@ -150,11 +174,149 @@ describe('validateServerVariables()', function () {
       }
     }`;
     const parsedInput = JSON.parse(inputString);
-  
-    expect(() => validateServerVariables(parsedInput, inputString, input)).to.throw('Not all server variables are described with variable object');
+
+    expect(() =>
```

Comment:
I know this is not your code but would be awesome if you could also change it to try-catch as I wrote in another comment 🙏🏼